website/blog/2024/07-05-launching-crawlee-python/index.md

```diff
@@ -0,0 +1,135 @@
+---
+slug: launching-crawlee-python
+title: 'Announcing Crawlee Python: Now you can use Python to build reliable web crawlers'
+description: 'Launching Crawlee for Python, a web scraping and automation libray to build reliable scrapers in Python fastly.'
+image: ./img/crawlee-python.webp
+author: Saurav Jain
+authorTitle: Developer Community Manager
+authorURL: https://github.com/souravjain540
+authorImageURL: https://avatars.githubusercontent.com/u/53312820?v=4&s=48
+authorTwitter: sauain
+---
+
+:::danger Testimonial from early adopters
+Crawlee for Python development team did a great job in building the product, it makes things faster for a Python developer. - [Maksym Bohomolov](https://apify.com/mantisus)
+:::
```

Comment:
```suggestion
:::danger Testimonial from early adopters

Crawlee for Python development team did a great job in building the product, it makes things faster for a Python developer. - [Maksym Bohomolov](https://apify.com/mantisus)

:::
```

also i am not entirely sure this should be the very first thing we put right after the heading, thoughts @netmilk @mnmkng?

also the `danger` type feels a bit weird but i guess that's ok

<img width="1041" alt="image" src="https://github.com/apify/crawlee/assets/615580/13e4d8cb-23c0-4666-9661-4efe450777fd">